Formula/l/lexido.rb

```diff
@@ -0,0 +1,21 @@
+class Lexido < Formula
+  desc "Innovative assistant for the command-line"
+  homepage "https://github.com/micr0-dev/lexido"
+  url "https://github.com/micr0-dev/lexido.git", tag: "v1.3.1", revision: "36f02d699f2709f0bd51a5f119e0670845b4ad9f"
+  license "AGPL-3.0-or-later"
```

Comment:
Where does this "or-later" come from? I could not tell from https://github.com/micr0-dev/lexido whether it's AGPL-3.0-only or AGPL-3.0-or-later.

Reply:
I tried to put “only” and got an error I then put “or later” and the error went away idk what's up with it

Reply:
CI would not complain about AGPL-3.0-only or AGPL-3.0-or-later; did you mean AGPL-3.0, which is deprecated?

It's not about trying which one works; it's about clarifying which one applies. It seems upstream has not made it clear, so can you ask them about this?

Reply:
No, I put only and done brew style lexido and it gave me errors that went away when. I put or later

Reply:
AGPL-3.0-or-later

Reply:
Is that documented anywhere?

Reply:
> Is that documented anywhere?

in the repo there is this 
[LICENSE](https://github.com/micr0-dev/lexido/blob/main/LICENSE)

and in the read me there is
```
License
Distributed under the GNU Affero General Public License v3.0. See LICENSE for more information.
```

Reply:
In the license though, it says:
>  Each version is given a distinguishing version number.  If the
Program specifies that a certain numbered version of the GNU Affero General
Public License "or any later version" applies to it, you have the
option of following the terms and conditions either of that numbered
version or of any later version published by the Free Software
Foundation.  If the Program does not specify a version number of the
GNU Affero General Public License, you may choose any version ever published
by the Free Software Foundation.

So that does not align with the comment about later versions since it only ever mentions v3. If upstream is fine with v3 or later I would just like that to be represented somewhere official.

Reply:
> In the license though, it says:
> 
> > Each version is given a distinguishing version number.  If the
> > Program specifies that a certain numbered version of the GNU Affero General
> > Public License "or any later version" applies to it, you have the
> > option of following the terms and conditions either of that numbered
> > version or of any later version published by the Free Software
> > Foundation.  If the Program does not specify a version number of the
> > GNU Affero General Public License, you may choose any version ever published
> > by the Free Software Foundation.
> 
> So that does not align with the comment about later versions since it only ever mentions v3. If upstream is fine with v3 or later I would just like that to be represented somewhere official.

would the repo owner need to put AGPL-3.0-or-later in the readme?

Reply:
It is also a comment on a [PR](https://github.com/micr0-dev/lexido/issues/19#issuecomment-2048652117) on the project, but I will make sure to put it somewhere else as well to make it clear